apps/rehabstod/src/pages/Patient/components/SelectRekoStatus/SelectRekoStatus.tsx

```diff
@@ -33,12 +33,11 @@ export function SelectRekoStatus({
   }
 
   return (
-    <SelectFilter
+    <Select
       disabled={disabled}
       value={rekoStatus?.status?.id || emptyRekoStatus?.id}
-      onChange={(id) => handleSetRekoStatus(id)}
-      options={rekoStatusTypes}
-      hideDefaultValue
```

Comment:
Is this functionality still working with the new changes or is it removed from requirements?

Reply:
It's still working, this was the only component that used `SelectFilter` that needed "hideDefaultValue", when changing it to this new `Select` from `@frontend/components` it's no longer needed and the prop could be deleted from `SelectFilter`